README.md

```diff
@@ -27,6 +27,8 @@ This repo houses summaries for various excitng works in the field of **Deep Lear
 
 - #### Human-level play in the game of Diplomacy by combining language models with strategic reasoning [[Paper](https://www.science.org/doi/epdf/10.1126/science.ade9097)][[Review](./summaries/CICERO.md)]
 
+- #### Photorealistic Text-to-Image Diffusion Models with Deep Language Understanding [[Paper](https://arxiv.org/abs/2205.11487)][[Review](./summaries/imagen.md)]
+
```

Comment:
Please add author names and publication venue here.

Reply:
Done